woocommerce/woocommerce
# Migrate `@woocommerce/number` to TS

### All Submissions:






### Changes proposed in this Pull Request:



Closes #33097.

This PR migrates `@woocommerce/number` to TS and fixes the wrong JSDoc type 


### How to test the changes in this Pull Request:

1. Run `pnpm nx build @woocommerce/number`
2. Observe that `packages/js/number/build-types` is generated
3. Should run `pnpm nx test @woocommerce/number` without errors

### Other information:




### FOR PR REVIEWER ONLY:
